doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc

```diff
@@ -0,0 +1,21 @@
+[id='create_archetype_project']
+= Creating a project using archetypes
+
+Archetypes are projects that are installed in Apache Maven repositories and contain a specific template structure. You can also generate parameterized versions of the project templates using archetypes. You can use archetypes to create a project in {CENTRAL}. When you create a project, it is added to the Git repository that is connected to your {PRODUCT} installation.
+
+.Prerequisites
+
+* You have created an archetype and listed in the {CENTRAL} *Settings*.
```

Comment:
I would be  more specific. You have created an archetype and added it to the Archetype section of the {CENTRAL} *Settings* page. 

BTW, do I create an archetype in BC or Maven (I really don't know). Mabye add a link;
For information about creating archetypes, see ...